src/main/java/org/zalando/nakadi/util/Aruha771MigrationHelper.java

```diff
@@ -0,0 +1,63 @@
+package org.zalando.nakadi.util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.zalando.nakadi.config.JsonConfig;
+import org.zalando.nakadi.domain.Subscription;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+/**
+ * Temporary class to be used for DB migration of ARUHA-771
+ */
+public class Aruha771MigrationHelper {
```

Comment:
I do not really like this class, because It is only "one time run" class, but it will be in our opened repository. Probably, it is better to move it closed repository.